charge-holy-weapon.lic

```diff
@@ -28,19 +28,19 @@ class HolyWeapon
   def check_charge_level(hw_settings)
     @em.wield_weapon(hw_settings['weapon_name'])
 
-    case bput("look my #{hw_settings['weapon_name']}", 'barely detectable', 'flickering', 'barely glowing', 'faintly', 'shining', 'emanating', 'blinding')
+    case DRC.bput("look my #{hw_settings['weapon_name']}", 'barely detectable', 'flickering', 'barely glowing', 'faintly', 'shining', 'emanating', 'blinding')
```

Comment:
Note to self, add link to wiki about these power levels, https://elanthipedia.play.net/Holy_weapon#Power_Levels